packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -7892,9 +7895,13 @@ export default class Meeting extends StatelessWebexPlugin {
    * Gets the time left in seconds till the permission token expires
    * (from the time the function has been fired)
    *
-   * @returns {number} time left in seconds
+   * @returns {object} containing permissionTokenTimeLeft, permissionExpiry, currentTime
    */
-  public getPermissionTokenTimeLeftInSec(): number | undefined {
+  public getPermissionTokenTimeLeftInSec(): {
```

Comment:
just to double check: you're changing a public JS-SDK API here, are we sure Cantina is not using it? 

Also, maybe the function should be renamed now, because it's not just a getter for the time left anymore

Reply:
Cantina is not using it, I checked it. I thought about changing name for this method but couldn't come up with something, suggestions are welcomed :)